Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/README.md

```diff
@@ -3253,4 +3256,275 @@
+</tbody>
+</table>
+</div>
+</div>
+<p> </p>
+<div class="cl-preview-section">
+<h5 id="command-example-18">Command Example</h5>
+</div>
+<div class="cl-preview-section">
+<pre>!esm-get-watchlists</pre>
+</div>
+<div class="cl-preview-section">
+<h5 id="context-example-10">Context Example</h5>
+</div>
+<div class="cl-preview-section">
+<pre>{
+    "EsmWatchlist": [
+        {
+            "ID": 3,
+            "IsActive": true,
+            "IsDynamic": false,
+            "Name": "Approved Addresses Watchlist",
+            "Type": "IPAddress",
+            "ValuesNumber": 16
+        },
+        {
+            "ID": 4,
+            "IsActive": true,
+            "IsDynamic": false,
+            "Name": "VIP Users Watchlist",
+            "Type": "UserIDSrc",
+            "ValuesNumber": 3
+        },
+        {
+            "ID": 5,
+            "IsActive": true,
+            "IsDynamic": false,
+            "Name": "Bad Hash Watchlist",
+            "Type": "Filename",
+            "ValuesNumber": 5
+        }
+    ]
+}
+</pre>
+</div>
+<div class="cl-preview-section">
+<h5 id="human-readable-output-17">Human Readable Output</h5>
+</div>
+<div class="cl-preview-section">
+<h3 id="watchlists-1">Watchlists:</h3>
+</div>
+<div class="cl-preview-section">
+<div class="table-wrapper">
+<table>
+<thead>
+<tr>
+<th>ID</th>
+<th>Type</th>
+<th>ValuesNumber</th>
+<th>IsDynamic</th>
+<th>IsActive</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>3</td>
+<td>IPAddress</td>
+<td>16</td>
+<td>false</td>
+<td>true</td>
+</tr>
+<tr>
+<td>4</td>
+<td>UserIDSrc</td>
+<td>3</td>
+<td>false</td>
+<td>true</td>
+</tr>
+<tr>
+<td>5</td>
+<td>Filename</td>
+<td>5</td>
+<td>false</td>
+<td>true</td>
+</tr>
+</tbody>
+</table>
+</div>
+</div>
+<h3 id="add-value-to-existing-watchlist">21. Add value to existing watchlist</h3>
+</div>
+<div class="cl-preview-section"><hr></div>
+<div class="cl-preview-section">
+<p>Adds values to an existing watchlist.</p>
+</div>
+<div class="cl-preview-section">
+<h5 id="base-command-16">Base Command</h5>
+</div>
+<div class="cl-preview-section">
+<p><code>esm-add-watchlist-value</code></p>
+</div>
+<div class="cl-preview-section">
+<h5 id="input-16">Input</h5>
+</div>
+<div class="cl-preview-section">
+<div class="table-wrapper">
+<table style="width: 749px;">
+<thead>
+<tr>
+<th style="width: 187px;"><strong>Argument Name</strong></th>
+<th style="width: 452px;"><strong>Description</strong></th>
+<th style="width: 101px;"><strong>Required</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td style="width: 187px;">id</td>
+<td style="width: 452px;">
+ID of the watchlist we want to add values to. ID can be retrieved with the "esm-get-watchlists" command.</td>
+<td style="width: 101px;">Required</td>
+</tr>
+<tr>
+<td style="width: 187px;">values</td>
+<td style="width: 452px;">
+Coma separated values of the data we want to add to the watchlist. Example: "1.1.1.1,2.2.2.2". Please add the correct type of values to correct watchlist type.</td>
```

Comment:
Please make sure the changes were made in the descriptions are applied in the readme too